llmfoundry/command_utils/data_prep/convert_delta_to_json.py

```diff
@@ -732,12 +735,38 @@ def fetch_DT(
     if dbsql is not None:
         dbsql.close()
 
-    # combine downloaded jsonl into one big jsonl for IFT
+    # Combine downloaded jsonl into one big jsonl for IFT.
     iterative_combine_jsons(
```

Comment:
wouldn't the output filename be empty before the iterative combine jsons command? Why do the validation after?

Reply:
I think `iterative_combine_jsons` writes to the output filename if the source dataset is across multiple files.

Reply:
(Meaning we can't do the validation earlier because the file at the path may not exist)